ui/app/pages/send/send-content/add-recipient/ens-input.component.js

```diff
@@ -53,16 +54,22 @@ export default class EnsInput extends Component {
     }
   }
 
-  // If an address is sent without a nickname, meaning not from ENS or from
-  // the user's own accounts, a default of a one-space string is used.
   componentDidUpdate (prevProps) {
     const {
       input,
     } = this.state
     const {
       network,
+      value,
     } = this.props
 
+    // Set the value of our input based on QR code provided by parent
+    if (input !== value && prevProps.value !== value) {
+      this.setState({ input: value })
+    }
+
+    // If an address is sent without a nickname, meaning not from ENS or from
```

Comment:
Do you know what this comment refers to? :thinking: I know you just moved it, but I can't figure out what it means. Better to delete it if it doesn't seem useful.